bin/install.sh

```diff
@@ -126,10 +126,10 @@ fi
 if [ -z "${REPOSITORY}" ]; then
   if [ "$WEB_UPGRADE" = false ]; then
     set -x
-    REPOSITORY=${1:-https://github.com/screenly/screenly-ose.git}
+    REPOSITORY=${1:-https://github.com/screenly/anthias.git}
```

Comment:
Just a minor comment. This could be capitalized &ndash; `Anthias`. (Apply to the rest of the changes.)

Reply:
Yeah i noticed that, but GitHub isn't case sensitive so i rather make it consistent.